src/inmanta/module.py

```diff
@@ -2976,18 +2976,24 @@ def _list_python_files(self, plugin_dir: str) -> list[str]:
         """
         Generate a list of all Python files in the given plugin directory.
         This method prioritizes .pyc files over .py files, uses caching to avoid duplicate directory walks,
-        includes namespace packages and excludes the model directory.
+        includes namespace packages and excludes the `model`, `files` and `templates` directories.
         """
         # Return cached results if this directory has been processed before
         if plugin_dir in self._dir_cache:
             return self._dir_cache[plugin_dir]
 
         files: dict[str, str] = {}
-        model_dir_path: str = os.path.join(plugin_dir, "inmanta_plugins", self.name, "model")
+        model_dir_path: str = os.path.join(plugin_dir, "model")
```

Comment:
where did the `"inmanta_plugins", self.name` part go?

Reply:
We saw with @sanderr that the plugin_dir was already containing the `"inmanta_plugins", self.name` part:
![Screenshot from 2024-08-28 13-24-08](https://github.com/user-attachments/assets/436208d0-f518-4c85-92fd-757ba216e80a)

Reply:
We believe it was bugged from the start and it doesn't seem to be covered by a test.